lib/diagnostic.rb

```diff
@@ -29,14 +29,14 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '@character is really a Sith Lord.'
 
```

Comment:
Remember how we did this on Monday?

using #{character} you in interpolate the string value into a string.